docs/zh/connector-v2/formats/canal-json.md

```diff
@@ -0,0 +1,115 @@
+# Canal 格式
+
+变更数据捕获格式:
+序列化模式、反序列化模式
+
+Canal是一款CDC（变更数据捕获）工具，能够实时捕获MySQL的数据变化并将其流式传输到其他系统中。Canal为变更日志提供了一种统一的格式，并支持使用 JSON 和 protobuf（Canal默认使用protobuf）进行消息的序列化
+
+SeaTunnel 能够解析 Canal 的 JSON 消息，并将其转化为 INSERT/UPDATE/DELETE 消息，进而输入到 SeaTunnel 系统中。这个特性在很多场景下都显得非常有用，例如:
+
+        将增量数据从数据库同步到其他系统
+        审计日志
+        数据库的实时物化视图
+        临时连接改变数据库表的历史记录等
```

Comment:
```suggestion
        关联维度数据库的变更历史，等等。
```
This refers to the translation of the [Flink document](https://nightlies.apache.org/flink/flink-docs-master/zh/docs/connectors/table/formats/canal/)